testsuite/tests/compiler-libs/test_longident.ml

```diff
@@ -56,32 +58,128 @@ let last_dot_apply = L.last
 val last_dot_apply : string = "foo"
 |}];;
 
-let parse_empty = L.parse ""
+type parse_result = { flat: L.t; spec:L.t; any_:L.t }
+let test specialized s =
+  { flat = L.parse_flat s;
+    spec = specialized (Lexing.from_string s);
+    any_ = PL.any (Lexing.from_string s)
```

Comment:
If we know that `spec` and `any_` should always be the same, I'm not convincing by showing them both in the expect-result and eyeballing that they are the same. I would rather have a field `any_correct = (PL.any ... = spec)` for which I could simply check that it is always `true` in the expected result.

Reply:
Done.